client/one-time-payment/react/.npmrc

```
@@ -0,0 +1 @@
+registry=https://registry.npmjs.org
```

Please add this to turn off the lock file
```suggestion
registry=https://registry.npmjs.org
package-lock=false
```